controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -628,6 +637,7 @@ func (r *StorageConsumerReconciler) reconcileCephClientRBDNode(
 	cephClientName string,
 	radosNamespaceName string,
 	additionalOwner client.Object,
+	currGeneration string,
```

Comment:
This function should not know anything about `curr`
```suggestion
        csiCephUserGeneration string,
 ```
 
 Please apply everywhere